### ISSUE TITLE
Expandable and collapsible pane UI

### ISSUE THREAD

**Issue by Michael-Grover** [accordproject/cicero-ui#119]
Expand button default and hover
https://projects.invisionapp.com/d/main#/console/16741009/370418065/preview
![image](https://user-images.githubusercontent.com/40569793/63553305-fbb69280-c507-11e9-9f8c-9b800d8677fa.png)

Cursor hovered on pane revealing a subtle border and the collapse button, which is otherwise not visible.
https://projects.invisionapp.com/d/main#/console/16741009/380412036/preview
![image](https://user-images.githubusercontent.com/40569793/63553335-12f58000-c508-11e9-960b-8880c9898ac5.png)

Cursor hovered over collapse button
https://projects.invisionapp.com/d/main#/console/16741009/380412037/preview
![image](https://user-images.githubusercontent.com/40569793/63553376-2bfe3100-c508-11e9-8d33-1eda05352549.png)

**Comment by irmerk** [comment]
Closed for [TSv2 Issue 80](https://github.com/accordproject/template-studio-v2/issues/80)
